app.py

```diff
@@ -52,69 +53,66 @@ def main():
         print("Accelerator: {}\n".format(obj_detect.accelerator))
         print("Labels:\n{}\n".format(obj_detect.labels))
 
-    tracker = edgeiq.CorrelationTracker(max_objects=5)
     fps = edgeiq.FPS()
     contraband_summary = ContrabandSummary()
 
+
+    def handle_detected_contraband(object_id, prediction):
+        print('Detected {}!'.format(prediction.label))
+        contraband_summary.update_contraband(prediction.label)
+        cv2.imwrite("{}_{}.jpeg".format(prediction.label, \
```

Comment:
This step could be done in `update_contraband` to keep the contraband-specific code isolated.

Reply:
Good idea! I've moved things around, I think they are now as you suggest.